### ISSUE TITLE
Staging Review finding: Overview graphic alt text ignored by Voiceover

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128410]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool Team 1
**Product:** Claim Status Tool
**Feature:** Document Submission Status

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On the Overview of the claim process imposter tab, the process graphic is ignored by iOS Voiceover. The user can't access the image via swipe. 

## Recommended action

This is a known iOS bug. To resolve, use `role="img"` and `aria-labelledby` on the `<svg>` 

```
<svg role="img" aria-labelledby="svgTitle" ...>
  <title id="svgTitle">A description of the image</title>
  <!-- SVG content -->
</svg>
```

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by dfitchett** [comment]
📸 **Screenshot of Accessibility Inspector verification:**
<img width="1191" height="1008" alt="Screenshot 2025-12-22 at 4 18 40 PM" src="https://github.com/user-attachments/assets/ee714ced-e04d-4d25-b851-4b5a669b6fbc" />

**Comment by amylai-va** [comment]
PO review completed ✅
